### PR TITLE
Add onboarding endpoint

### PR DESCRIPTION
**Pull request by kingel**

### CODE REVIEW

File: mollie/api/resources/onboarding.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class Onboarding(Base):
+
+    def get_resource_object(self, result):
+        return OnboardingObject(result, self.client)
+
+    def get(self, onboarding_id, **params):
+        if not onboarding_id or not onboarding_id == 'me':
+            raise IdentifierError(
+                "Invalid refund ID: '{id}'. A refund ID should be 'me'.".format(
```
Comment (whyscream) [inline mollie/api/resources/onboarding.py]: refund?

File: tests/test_onboarding.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    assert onboarding.name == 'Mollie B.V.'
+    assert onboarding.signed_up_at == '2018-12-20T10:49:08+00:00'
+    assert onboarding.status == 'completed'
+    assert onboarding.can_receive_payments is True
+    assert onboarding.can_receive_settlements is True
+
+
+def test_create_onboarding(client, response):
+    """Update onboarding.."""
```
Comment (whyscream) [inline tests/test_onboarding.py]: create is not update

File: mollie/api/objects/onboarding.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    @property
+    def name(self):
+        return self._get_property('name')
+
+    @property
+    def signed_up_at(self):
+        return self._get_property('signedUpAt')
+
+    @property
+    def status(self):
```
Comment (whyscream) [inline mollie/api/objects/onboarding.py]: When a field like `status` as some predefined values, we add convenience methods and constants for them: see https://github.com/mollie/mollie-api-python/blob/982a1a63e8217c6c1fe03c52fde6e09e3e544f9d/mollie/api/objects/subscription.py#L21

File: tests/test_onboarding.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    response.patch('https://api.mollie.com/v2/onboarding/me', 'empty', 204)
+
+    data = {
+        'profile': {
+            'categoryCode': '6012'
+        }
+    }
+
+    onboarding = client.onboarding.update(resource_id='me', data=data)
```
Comment (whyscream) [inline tests/test_onboarding.py]: https://docs.mollie.com/reference/v2/onboarding-api/submit-onboarding-data Documents only POST, not PUT/PATCH.

Comment (kingel) [reply tests/test_onboarding.py]: I changed this

Comment (whyscream) [reply tests/test_onboarding.py]: Could you also update the test name and the docstring? you are creating data, not updating it, I think.

Comment (kingel) [reply tests/test_onboarding.py]: i updated this

**Pull request merged**